src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -125,6 +125,11 @@ public static class Parser
         private static readonly Regex YearInTitleRegex = new Regex(@"^(?<title>.+?)(?:\W|_)?(?<year>\d{4})",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
+        //Handle Exception Release Groups that don't follow -RlsGrp; Manual List
+        //First Group is groups whose releases end with RlsGroup) or RlsGroup]  second group (entries after `(?=\]|\))|`) is name only...BE VERY CAREFUL WITH THIS, HIGH CHANCE OF FALSE POSITIVES
+        private static readonly Regex ExceptionReleaseGroupRegex = new Regex(@"(?<releasegroup>(Tigole|Joy|YIFY|YTS.MX|FreetheFish|afm72)(?=\]|\))|KRaLiMaRKo|E\.N\.D)",
```

Comment:
Can you add some false positive tests for `joy` at least, to make sure it doesn't pickup when we don't want it to if there is no release group.

Reply:
added, passed locally, pushed